.github/workflows/php-unit-tests.yml

```diff
@@ -101,6 +101,17 @@ jobs:
       - name: Install WP tests
         run: ./bin/install-wp-tests.sh wordpress_test root root localhost ${{ matrix.wp-version }} ${{ matrix.wc-versions }}
 
+      - name: Prepare node
```

Comment:
@jamesmorrison can we revert all changes to this file. It appears these changes are the ones that have made the PHPunit tests workflow stop working, where as in the previous commit unit tests would run but fail. Once we have the unit tests working again we can see how to resolve the specific issue you faced.

Reply:
This has been reverted